packages/client/package.json

```diff
@@ -71,7 +71,7 @@
     "tailwind-merge": "^1.13.2",
     "tailwindcss": "^3.3.2",
     "ts-node": "10.9.1",
-    "vite": "5.0.9",
+    "vite": "5.1.3",
```

Comment:
gotta revert probably. But honestly we should update. There are security issues with 5.0.9

Reply:
let's do this in a separate PR and deploy to QAT